### ISSUE TITLE
Full text search should include notes with the searched text in a tag

### ISSUE THREAD

**Issue by mlncn** [standardnotes/web#191]
That is, if i have a note titled "A People's History of the United States" with content "Excellent book; kept in our social justice library" and tagged "USA"— when i search for "USA" i expect this note to come up even though "USA" is not in the title nor the content, just the tag.

(Not sure if #64 was ever interpreted this way, but in any case this does not currently work on the hosted version at standardnotes.org.)

**Comment by mobitar** [comment]
This is interesting. I'm trying to understand why this would be desired, generally. Why not just select the USA tag and search in there? Can you give some other examples where this would be useful?

**Comment by mlncn** [comment]
Because i have to remember what i tag it.

Basically, i think that tagging by keyword and searching by keyword are very common use cases.  But no one should have to remember if the keyword was a formal tag or just in the title or text.

**Comment by vitormv** [comment]
i agree with @mobitar . Makes a lot of sense to search also in the tags as well as note title and body.
